src/admin/mod.rs

```diff
@@ -287,6 +288,13 @@ async fn handle_config_dump(dump: WorkloadInformation, _req: Request<Body>) -> R
         .unwrap()
 }
 
+async fn handle_help(_req: Request<Body>) -> Response<Body> {
+    Response::builder()
+        .status(hyper::StatusCode::OK)
+        .body("/healths/ready\t\t\tprint server state\n/debug/pprof/profile\t\tprint the profile data\n/debug/gprof/profile\t\tprint the gprof\n/debug/gprof/heap\t\tprint the heap profile\n/quitquitquit\t\t\texit the ztunnel\n/config_dump\t\t\tprint the workload information\n/help\t\t\t\tprint the command list\n".into())
```

Comment:
Could you convert this to a readable multi-line string?

Reply:
> Could you convert this to a readable multi-line string?
I am working on it, but my limited rust skill make it looks ugly, will send it out when I fix it.